userguide/advanceduse/dav.rst

```diff
@@ -14,22 +14,48 @@ These instructions work for CalDAV as well.
 1) Follow this `guide <https://docs.ubports.com/en/latest/userguide/advanceduse/adb.html>`_ to activate Developer Mode and a connection either through ADB or SSH.
 
 
-2) Download this `script <https://gist.github.com/bastos77/0c47a94dd0bf3e394f879c0ff42b7839>`_ (let's call it dav.sh) and edit the following variables: 
+2) OPTION A: 
 
-* Server side: CAL_URL, **CONTACTS_URL**, USERNAME, PASSWORD (of your ownCloud/nextCloud/baikal/SOGO/… server)
-* CONTACT and CALENDAR _ NAME / VISUAL_NAME / CONFIG_NAME (it's more cosmetic)
-* CRON_FREQUENCY (for the frequency of synchronization)
-* Line 61: write ``sudo sh -c "echo '$COMMAND_LINE' > /sbin/sogosync"`` , instead of ``sudo echo "$COMMAND_LINE" > /sbin/sogosync``, to avoid a "Permission denied" error.
+    2.1    Download this `script <https://gist.github.com/bastos77/0c47a94dd0bf3e394f879c0ff42b7839>`_ (let's call it dav.sh) and edit the following variables: 
```

Comment:
please check the indentation. it is formatted like a quotation https://ci.ubports.com/job/docs.ubports.com/job/PR-508/7/artifact/_build/html/userguide/advanceduse/dav.html which is confusing

Also, you might want to consider using autonumbered lists https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html Actually, what do you think about getting rid of the top layer of the numbered list and just writing two subsections for the two options. And giving a short intro sentence statting that there are two options (can you say smt about pro's and con's - don't worry if you cannot, but if you have it clear in your mind it would be helpful to share)

Actually, the existing intro doesn't really fit anymore and should be updated:

> There is a script you can run

While you are at it, could you do me a favor and also update the paragraph before:

> Note: CaDAV sync can also ... 

Let's start from the normal/positive case before we get to the negative/exception/advanced/manual stuff. Smt like 

CalDAV can be set up using the calendar app. Open it ....